buildSrc/src/main/kotlin/com.hedera.hashgraph.maven-publish.gradle.kts

```diff
@@ -90,20 +87,17 @@ signing {
     sign(publishing.publications.getByName("maven"))
 }
 
-tasks.register("release-maven-central") {
+tasks.register("releaseMavenCentral") {
     group = "release"
-    dependsOn(tasks.named<Task>("publishMavenPublicationToSonatypeRepository"))
+    dependsOn(tasks.named("publishMavenPublicationToSonatypeRepository"))
 }
 
-tasks.register("release-maven-central-snapshot") {
+tasks.register("releaseMavenCentralSnapshot") {
     group = "release"
-    dependsOn(tasks.named<Task>("publishMavenPublicationToSonatypeSnapshotRepository"))
+    dependsOn(tasks.named("publishMavenPublicationToSonatypeSnapshotRepository"))
 }
 
 java {
-    if (tasks.getByName("release-maven-central").enabled ||
-            tasks.getByName("release-maven-central-snapshot").enabled) {
```

Comment:
This code had no effect. "enabled == false" is never the case. For that you would need some logic to explicitly disable the task(s).

It does **not** tell if the task is part of the current build (or not). If this was the intention.

Reply:
Agreed, this was residual code which was inadvertently left when trying to condition on only building javadoc and source jars if releasing.